rootfs/init.sh

```diff
@@ -3,4 +3,8 @@
   find /cloudposse/jenkins/plugins.d/ -type f -exec cat {} \; | \
   xargs /usr/local/bin/install-plugins.sh
 
+[ -d /cloudposse/jenkins/jobs.d/ ] && \
+  ls  /cloudposse/jenkins/jobs.d | xargs -I {} mkdir -p /var/jenkins_home/jobs/{}  && \
```

Comment:
> also don't like that the ls is not restricted to directories; should probably use find
It should be restricted to files really.

> can we maybe just use just use cp -an /cloudposse/jenkins/jobs.d/ /var/jenkins_home/jobs/?
Nope. The solution expect flat files in that directory. Because k8s secretes / configmaps can not support implicit directory structures. 
So for easy use it makes such map

/cloudposse/jenkins/jobs.d/job-1 -> /var/jenkins_home/jobs/job-1/config.xml
/cloudposse/jenkins/jobs.d/job-2 -> /var/jenkins_home/jobs/job-2/config.xml

To make secret/configmap to bind directory such way is very easy.